src/js/map/maps.js

```diff
@@ -278,15 +292,8 @@ export class MapControl extends Observable {
                 .addTo(self.map);
 
             if (self.zoomToPosition() && !alreadyZoomed) {
-                try {
-                    self.map.flyToBounds(layer.getBounds(), {
-                        animate: true,
-                        duration: 0.5 // in seconds
-                    });
-                    alreadyZoomed = true;
```

Comment:
@adieyal  `alreadyZoomed` was only changed when the comannd did not fail. 
but now it is always changed to `true` is that desired?

Reply:
I have updated the code in the original PR

https://github.com/OpenUpSA/wazimap-ng-ui/pull/164